changelogs/internal/newsfragments/1601.clarification

```diff
@@ -0,0 +1 @@
+Improve the layout of tables on desktop.
```

Comment:
```suggestion
Improve the layout of tables on desktop displays. Contributed by @not-my-profile.
```

Reply:
I don't really like my work being attributed to my GitHub username :/

Reply:
Oh, sorry. I can remove this.

Reply:
would you prefer an alternative attribution?

Reply:
Yeah: Martin Fischer.

Reply:
Done: https://github.com/matrix-org/matrix-spec/commit/f5035b8e0271649816ab35dc12e9a5cc5e7783c3

Reply:
Thanks <3